index.js

```diff
@@ -1,6 +1,8 @@
+'use strict'
 const helmet = require('helmet');
 const express = require('express');
-const app = module.exports = express();
+const app = express();
+module.exports = app
```

Comment:
Does it still work if we move this to the bottom of index.js? I think it's more node idiomatic that way.

Reply:
it doesn't work if it's below 
let db = require('./db/database_mongo');